integration-tests/docker/Dockerfile

```diff
@@ -1,37 +1,5 @@
-# This is intended to be a temporary unblocker for Travis CI
-# We should revert this when ppa:webupd8team/java repo maintainers fix the issue shown here: https://github.com/druid-io/druid/pull/4970
-# Or if we stick to using a non-base Ubuntu image, the custom image should reside in an org repo and not an individual repo
-# https://hub.docker.com/r/jonweiimply/ubuntu-j8/
-FROM jonweiimply/ubuntu-j8
-
-# MySQL (Metadata store)
-RUN apt-get install -y mysql-server
-
-# Supervisor
-RUN apt-get install -y supervisor
-
-# Zookeeper
-RUN wget -q -O - http://www.us.apache.org/dist/zookeeper/zookeeper-3.4.6/zookeeper-3.4.6.tar.gz | tar -xzf - -C /usr/local \
-      && cp /usr/local/zookeeper-3.4.6/conf/zoo_sample.cfg /usr/local/zookeeper-3.4.6/conf/zoo.cfg \
-      && ln -s /usr/local/zookeeper-3.4.6 /usr/local/zookeeper
-
-# Kafka
-# Match the version to the Kafka client used by KafkaSupervisor
-RUN wget -q -O - http://www.us.apache.org/dist/kafka/0.10.2.0/kafka_2.11-0.10.2.0.tgz | tar -xzf - -C /usr/local \
-      && ln -s /usr/local/kafka_2.11-0.10.2.0 /usr/local/kafka
-
-# Druid system user
-RUN adduser --system --group --no-create-home druid \
-      && mkdir -p /var/lib/druid \
-      && chown druid:druid /var/lib/druid
-
-# clean up time
-RUN apt-get clean \
-      && rm -rf /tmp/* \
-                /var/tmp/* \
-                /var/lib/apt/lists \
-                /root/.m2
-
+# Base image is built from integration-tests/docker-base in the Druid repo
+FROM imply/druiditbase
```

Comment:
It would be nice to eventually pull from a druid-io source, [ECR](https://aws.amazon.com/ecr/), [GCR](https://cloud.google.com/container-registry/), [Docker Hub](https://hub.docker.com/), and [Quay](https://quay.io/) are all options. Is there any pro or con to any of them that folks know about?

Reply:
Docker Hub is built into docker so that's a pro in favor of that one. It's also free. It's pretty bare bones but should be good enough for what we're doing.

There is already a "semi-official" https://hub.docker.com/r/druidio/ that @xvrl set up: https://groups.google.com/d/topic/druid-development/RCLPKrBEMr0/discussion

Reply:
Definitely makes sense to me to move this to druidio on docker hub if that repo is made "official" instead of "semi-official" (probably just meaning more druid people should have push access).

Reply:
I meant for it to become the official one, @jon-wei is the only one that gave me his login. Maybe once I have all the PMC members in there, it can be christened as the official one? @gianm @fjy @himanshug @drcrallen @cheddar @nishantmonu51

Reply:
@xvrl that sounds good to me; my docker hub id is "gianm"

Reply:
@xvrl mine is "himanshu01"

Reply:
@gianm @himanshug you should have access now.

Reply:
thanks @xvrl

Reply:
@xvrl thanks